tutorials/get-started-kubernetes-storage/index.mdx

```diff
@@ -0,0 +1,247 @@
+---
+meta:
+  title: Getting started with Kubernetes Part 4 - Storage
+  description: This is the fourth in a series of tutorials accompanying our video demonstrations on getting started with Kubernetes. Part 4 explains how to add storage to your cluster.
+content:
+  h1: Getting started with Kubernetes Part Part 4 - Storage
+  paragraph: This is the fourth in a series of tutorials accompanying our video demonstrations on getting started with Kubernetes. Part 4 explains how to add storage to your cluster.
+tags: Kubernetes k8s managed-kubernetes services kapsule kubectl storage volume block-storage stateful-set persistent-volume persistent-volume-claim 
+categories:
+  - kubernetes
+dates:
+  validation: 2024-01-29
+  posted: 2024-01-29
+---
+
+This tutorial accompanies the fourth [video demonstration](https://www.youtube.com/watch?v=OAR_3Fuwtw0) in our series to help users get started with Kubernetes. We walk you through Kubernetes fundamentals for beginners. In this installment, we show you how to add storage to your [Scaleway Kubernetes Kapsule](https://www.scaleway.com/en/kubernetes-kapsule/).  
+
+First, we address the question of why we might need to add storage to our cluster, comparing **persistent** storage with **system** storage, and digging into **stateless** vs. **stateful** applications. We then show how to create a stateful application in a cluster via a **StatefulSet object**, which provisions persistent storage via a PersistentVolumeClaim, which in turns provisions and manages storage using the Scaleway Block Storage product.
+
+Previous videos in this series covered the basics of [containers and Docker](/tutorials/get-started-containers-docker/), [deploying an app with Kapsule](/tutorials/get-started-deploy-kapsule/), and [adding a Load Balancer](/tutorials/get-started-kubernetes-loadbalancer/).
+
+<Macro id="iam-requirements" />
```

Comment:
You might need to rebase and rewrite the requirements

Reply:
@RoRoJ I updated the requirements part (see below), not sure if it needs a rebase as this is a new page